app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -36,17 +42,25 @@ class ExportWorkflowsDialog extends React.Component {
     getWorkflowsInOrder(project, { fields: 'display_name' })
       .then((workflows) => {
         this.setState({ workflows, loading: false });
-      });
-  }
 
-  toggleExport() {
-    if (this.workflowList.selectedIndex >= 0) {
-      this.setState({ workflowSelected: true });
-    }
+        // TODO: load the exports for each workflow
+        workflows.forEach((wf) => {
+          wf.get('classifications_export')
+            .then((media) => {
+              const mediaState = this.state.media;
+              mediaState[wf.id.toString()] = media[0];
+              this.setState({ media: mediaState });
+            })
+            .catch(() => {
+              // this is gonna happen, oh well
```

Comment:
We fixed the error handling in the json-api-client so you can appropriately handle different error codes. So this could be something like `if (error.status !== 404) doSomething`. See example with the classifier: https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/classify.cjsx#L281-L288